src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -48,7 +48,7 @@ class SignInModal extends React.Component {
             <h1>Sign in to {siteName}</h1>
           </div>
         </div>
-        <div className="row hide-for-medium-up mobile-explanation">
+        <div className="row xsmall-screen:vads-u-display--block small-screen:vads-u-display--none mobile-explanation">
```

Comment:
Do you need the `xsmall-screen` one? Also, it seems like it might be `medium-screen:vads-u-display--none` instead of `small-screen` for the second one?